pkg/job/runner.go

```diff
@@ -76,6 +76,7 @@ func (j *Job) Run(ignoreSidecar bool) error {
 	}
 	running, err := j.RunJob()
 	if err != nil {
+		log.Error(err)
```

Comment:
This is useful for debugging.

*before*
I couldn't understand why kube-job tries to remove job when it hasn't even started yet.
```
$ kube-job run --ignore-sidecar --config=/path/to/config --template-file=/path/to/job.yml --container="foo" -v
INFO[0000] Using config file: /path/to/config
INFO[0000] Ignore sidecar containers
INFO[0000] Removing the job: this-is-long-label-named-job-hello-hello-hello-4cb7511ae74742eed134e86c5ece1375
FATA[0000] jobs.batch "this-is-long-label-named-job-hello-hello-hello-4cb7511ae74742eed134e86c5ece1375" not found
```

*after*
```
$ kube-job run --ignore-sidecar --config=/path/to/config --template-file=/path/to/job.yml --container="foo" -v
INFO[0000] Using config file: /path/to/config
INFO[0000] Ignore sidecar containers
ERRO[0000] Job.batch "this-is-long-label-named-job-hello-hello-hello-451746b71378ea96346fa9924b18ecb5" is invalid: spec.template.labels: Invalid value: "this-is-long-label-named-job-hello-hello-hello-451746b71378ea96346fa9924b18ecb5": must be no more than 63 characters
INFO[0000] Removing the job: this-is-long-label-named-job-hello-hello-hello-4cb7511ae74742eed134e86c5ece1375
FATA[0000] jobs.batch "this-is-long-label-named-job-hello-hello-hello-4cb7511ae74742eed134e86c5ece1375" not found
```